### PR TITLE
fix: bring back url phishing warning

### PR DESCRIPTION
**Pull request by moodysalem**

### CODE REVIEW

File: src/pages/App.tsx [hunk 1]
```diff
@@ -52,12 +54,12 @@ const BodyWrapper = styled.div`
   `};
 `
 
-const HeaderWrapper = styled.div`
+const HeaderWrapper = styled.div<{ urlWarningVisible?: boolean }>`
```
Comment (moodysalem) [inline src/pages/App.tsx:57]: required isn't it?

File: src/components/Header/URLWarning.tsx [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+          <code style={{ padding: '0 4px', display: 'inline', fontWeight: 'bold' }}>app.uniswap.org</code>
+        </Trans>
+      </div>
+      <StyledClose size={12} onClick={toggleURLWarning} />
+    </PhishAlert>
+  ) : (
+    <PhishAlert>
+      <div style={{ display: 'flex' }}>
+        <AlertTriangle style={{ marginRight: 6 }} size={12} />
+        <Trans>
```
Comment (moodysalem) [inline src/components/Header/URLWarning.tsx:45]: only this is different, the rest is the same, move the ternary in

File: src/pages/App.tsx [hunk 3]
```diff
@@ -52,12 +54,12 @@ const BodyWrapper = styled.div`
...
   ${({ theme }) => theme.flexRowNoWrap}
   width: 100%;
   justify-content: space-between;
   position: fixed;
-  top: 0;
+  top: ${({ urlWarningVisible }) => (urlWarningVisible ? '20px' : 0)};
```
Comment (moodysalem) [inline src/pages/App.tsx:62]: this bit is odd, the warning may be larger if the text wraps

**Comment by aballerr** [comment]
confirmed with Moody this can be closed

**Pull request closed without merging**